modules/core/presentation/templates/layouts/authenticated.templ

```diff
@@ -214,7 +214,7 @@ templ Authenticated(props AuthenticatedProps) {
 		Footer: SidebarFooter(pageCtx),
 	}
 	}}
-	@Base(props.Title) {
+	@Base(&BaseProps{Title: props.Title, WebsocketURL: "/ws"}) {
```

Comment:
Let's embed `BaseProps` into `AuthenticatedProps`. This will come in handy in the next task.
And in Authenticated layout set defaults.
```go
if props.WebsocketURL == "" {
  props.WebsocketURL = "/ws"
 }
```